app.py

```
@@ -0,0 +1,136 @@
+            "만약 문서가 없으면 질문에 적합한 답변만 해줘."
+        )
+
+        full_prompt = f"{prompt_to_use}\n\n문서 참고 내용:\n{context_text}\n\n질문: {req.message}"
+
+        model = genai.GenerativeModel("gemini-1.5-flash")
+        response = model.generate_content(full_prompt)
+        logging.info("[LLM 노드] Gemini 응답 완료")
+
+        return {"answer": response.text}
+
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)